sotodlib/coords/pointing_model.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from .. import core
+from .helpers import _valid_arg
+
+DEG = np.pi / 180
+
+
+def apply_pointing_model(tod):
+    """Redefines tod.boresight using pointing model parameters.
+
+    Args:
+        tod: AxisManager
+
+    Returns:
+        tod.boresight will be updated.
```

Comment:
This doesn't return anything, currently.  Just None.  Mention the effect of the function (to modify tod.boresight) in the main description, not as a return parameter.

Or better, you could make it return the _b AxisMan that you create.  That would be useful, because we could imagine this (in the future) being something that computes the new boresight but does not _necessarily_ load it into tod yet.